apt_repo/__init__.py

```diff
@@ -4,8 +4,7 @@
 import os
 import posixpath
 import re
-import urllib.error
-import urllib.request as request
+import requests
```

Comment:
Requests is an external package. I think the PR should contain changes in the package requirements

Reply:
done here https://github.com/3nids/python-apt-repo/commit/e09f4aa000db0329da79b550a623e72da0d96687